implementation/core/src/main/java/io/smallrye/faulttolerance/core/retry/CompletionStageRetry.java

```diff
@@ -31,10 +33,8 @@ public CompletionStageRetry(
         return doRetry(ctx, 0, runningStopwatch, null);
     }
 
-    public CompletionStage<V> doRetry(InvocationContext<CompletionStage<V>> target,
-            int attempt,
-            RunningStopwatch stopwatch,
-            Throwable latestFailure) {
+    public CompletionStage<V> doRetry(InvocationContext<CompletionStage<V>> target, int attempt,
```

Comment:
was it the auto-formatter? :(

Reply:
Looks like unintended and unnecessary change.

Reply:
But n.b. given the time press its moot to me.

Reply:
This might have actually been me :-) The one line per param format is so inefficient with screen space :-)